python/py-gmpy/Portfile

```diff
@@ -5,7 +5,7 @@ PortGroup           python 1.0
 
 name                py-gmpy
 version             1.17
-maintainers         alluvialsw.com:md14-macports openmaintainer
+maintainers         @mndavidoff openmaintainer
```

Comment:
Usually we would want both their email _and_ their github handle.

Reply:
Added back my email. Thanks!